### ISSUE TITLE
Priority- add 'a Data accuracy' info box on the main page

### ISSUE THREAD

**Issue by polly64** [tomalrussell/colouring-london#277]
Hi Tom data, data accuracy is a big question people are asking about. Several  have raised the issue of needing a clear statement that checking data accuracy is the responsibility of the user as information could potentially be used in planning applications etc. and also that UCL cannot be responsible for data errors.

I suggest that we put a 'data accuracy info ' button up on the main screen asap which can both click and  hover on to bring up the following:

' Our data are crowdsourced from multiple sources and may contain errors. We are unable to vouch for the accuracy of individual entries. However we will be adding a number of features - including 'Source', 'Edit history' and 'Crowd Verification' buttons - to assist you in checking reliability and judging how suitable the data are for your intended use'.

**Comment by dominijk** [comment]
@polly64 The logical place (For me) is that this message is in the first welcome popup you get on visiting the site as mock up below. We can then either add a new link in the top bar that displays another popup or link to page on WIX site or edit the top toolbar to include the accuracy in one of the existing links as options below;

Data Categories & Accuracy
OR
Data Ethics & Accuracy

One we can review on Tuesday

<img width="1282" alt="Screen Shot 2019-07-12 at 14 27 44" src="https://user-images.githubusercontent.com/6041913/61131497-4d074700-a4b1-11e9-9bae-ddec04607ef9.png">

**Comment by polly64** [comment]
lets discuss tues - i actually think the short intro is necessary as people won't want to read much- however agree need v visible.  i'd prefer it to be on a) data sign up AND b) data ethics page, AND c) under building categories AND  d) and as popup on view and edit pages-  so real belt and braces stuff but just less full on as you enter the site


Polly Hudson

Doctoral researcher (EPSRC DTG)

Bartlett Centre for Advanced Spatial Analysis

University College London

90 Tottenham Court Road W1T 4TJ




________________________________

**Comment by dominijk** [comment]
from #306 

I asked for a wording from UCL for a disclaimer and this has been sent - it can also be used as a supplement to the text I have already drafted. So I think we should keep the drafted text on the screen so can be accessed at any time and as it is less aggressive - (this can also come up large on first visit) but also link this to the full UCL disclaimer as below. Both should also be included separately on the sign up page and also on the download page . It is very important that people understand it is their responsibility for checking the accuracy of the data for their applications not only from our legal point of view but also as a basic issue of getting people have to check 'facts'.

The UCL disclaimer sent to us reads as follows: the data are provided "as is", without warranty of any kind, express or implied, including but not limited to the warranties of merchantability, accuracy, fitness for a particular purpose and non-infringement. In no event shall UCL be liable for any reliance that you place on or how you use the data nor any claim, damages or other liability, whether in an action of contract, tort or otherwise, arising from, out of or in connection with the data or the use or other dealings in the data'.
Capturing comment from 
I also notice on the GLA's London Development database they have this .

Disclaimer
Whilst every effort has been made to ensure the accuracy of the data, the GLA does not accept any responsibility for any errors or omissions it may contain. The data on this site remains the property of the London Local Authorities, who are required to keep it up to date and accurate.The use of the London Development Database public website is subject to the same Terms & Conditions and Privacy Policy as LONDON.GOV.UK.

We obviously aren't required to keep the CL data accurate

**Comment by tomalrussell** [comment]
Link to https://colouring.london/data-accuracy.html

**Comment by mz8i** [comment]
What is the final form we would like to have this in?
- where should the info box appear? On the main welcome page only?
- I assume it should be possible to dismiss it?
- what text should it contain?
- should it link to other pages, like the data accuracy statement Tom linked to above?

**Comment by polly64** [comment]
Hi, it's already in a number of places which is great so don't think needs a link. As this question is asked so many times agree  it is  good to test thison the landing page - as image above -( rather than move to the top of the view/edit map page under a 'data accuracy button')  .   

However I just think we should adjust and shorten the text slightly so the last para reads 

'We collect our building data from many sources. Though we are unable to vouch for their accuracy, we are currently experimenting with a range of features including'data source', 'edit history', and  'entry verification',  to assist you in checking reliability and judging how suitable the data are for your intended use.

Of course any thoughts let me know

**Comment by mz8i** [comment]
Right, I missed the fact that the image above contains the modified text. I think it makes sense to just add it there rather than add another UI element. I will then release a simple change adding the latest text to a paragraph on the welcome screen.

**Comment by mz8i** [comment]
Polly suggested some edits to the Welcome page. Below is the result of applying the changes:
- merging the first two paragraphs into one, shorter paragraph
- switching the data accuracy paragraph with the attribution paragraph
- substituting links on the welcome page with static text
Resulting text:
```
Colouring London is a knowledge exchange platform collecting information on every building in London, to help make the city more sustainable. We’re building it at The Bartlett Centre for Advanced Spatial Analysis, University College London. We're still at an early stage of development. Please help us by providing feedback on the site as we create it!

We collect our building data from many sources. Though we are unable to vouch for their accuracy, we are currently experimenting with a range of features including 'data source', 'edit history', and 'entry verification', to assist you in checking reliability and judging how suitable the data are for your intended use.

All of the data we collect is made openly available. Please read our data ethics policy and credit Colouring London if you use or share our maps or data.
```
Screenshot:
![image](https://user-images.githubusercontent.com/36160844/68782874-5c062d80-063a-11ea-81c6-3519ffb03866.png)

@tomalrussell @polly64 please comment on the proposed text. My comment would be that without the links maybe the content of the attribution paragraph are less clear - it's not obvious where the data is available, and where is the data ethics policy.

**Comment by polly64** [comment]
Sure tom, 
Maciej can we 
a) highlight links in another greyish colour so they don't stand out too much? 
b) Also can you change last line to . 'We just ask you to credit Colouring London and read our data ethics policy to use or share our data and maps'  Happy with this line?-think just makes it a bit more friendly.
c) remove exclamation mark at end of first para.

**Comment by mz8i** [comment]
Updated view:
![image](https://user-images.githubusercontent.com/36160844/68793330-4bab7e00-064d-11ea-9974-7dcd549626c9.png)

**Comment by polly64** [comment]
Hi Maciej sorry we just need a few changes I realise
Para 1 - End first  para after ' sustainable' to  change to' We're developing it at University College London. Can you help us? We're looking for volunteers of all ages and abilities to help test the site and colour the buildings in'. 

Para 2 change (ist line): 
'Our building data comes from many different sources.'

Para 3 change 'All of the data we collect is' to 'All data we collect are' .  End of para change to  'when using or sharing our data, maps or code' . Do you both agree might be good to add word 'code' here with a link to Github to stimulate experimentation by other cities?

**Comment by mz8i** [comment]
Closed by #498
